Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -339,8 +340,9 @@ void AODProducerWorkflowDPL::fillTrackTablesPerCollision(int collisionID,
             const double massZ = o2::track::PID::getMass2Z(trackPar.getPID());
             const double energy = sqrt((massZ * massZ) + (extraInfoHolder.tofExpMom * extraInfoHolder.tofExpMom));
             const double exp = extraInfoHolder.length * energy / (cSpeed * extraInfoHolder.tofExpMom);
-            extraInfoHolder.trackTime = static_cast<float>((tofMatch.getSignal() - exp) * 1e-3 - interactionTime); // tof time in \mus, FIXME: account for time of flight to R TOF
-            extraInfoHolder.trackTimeRes = 200e-3;                                                                 // FIXME: calculate actual resolution (if possible?)
+            // extraInfoHolder.trackTime = static_cast<float>((tofMatch.getSignal() - exp) * 1e-3 - interactionTime); // tof time in \mus, FIXME: account for time of flight to R TOF
```

Comment:
Why has the unit changed by 3 orders of magnitude and the comment stays the same: ` \mus` ?

Reply:
This is a fix, the time was stored in `ns` before and the comment was inconsistent. Now we store both trackTime and trackTimeRes with the same units

Reply:
in the comment you still have mus. Could you write "ns"? I guess this was the comment of @jgrosseo

Reply:
Sorry, you move to mus everything (I was mislead by the part commented)